Marsiple/App/Albums/AlbumTableViewCell.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  AlbumTableViewCell.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 19/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import UIKit
+
+class AlbumTableViewCell: UITableViewCell {
+    private var images = [UIImage]()
```

Comment:
For now, this is okay. But the `Cell`, that represents the `View` layer, shouldn't have any business logic at all. It should just be a plain-old, stupid `View` that only knows how to display data it's given.